light-client/tests/light_client.rs

```diff
@@ -152,13 +153,16 @@ fn run_bisection_test(tc: TestBisection<LightBlock>) {
     let trusting_period = tc.trust_options.period;
     let now = tc.now;
 
+    // FIXME: What should this be, and where should it be configured?
```

Comment:
in Go, default is 10sec. once we switch to the proposer based timestamps, it will probably be a consensus parameter